.github/dependabot.yml

```
@@ -10,6 +10,10 @@ updates:
         patterns:
           - "trame"
           - "trame-*"
+      sphinx:
+        patterns:
+          - "sphinx"
+          - "sphinx-*"
```

What about trying to clob together all documentation building specific items?  My suggestion below, but maybe missing a few too.

```suggestion
          - "*sphinx*"
          - "pypandoc"
```

---

Thanks! Let's try and see what's happen.